mathics/eval/nevaluator.py

```diff
@@ -34,15 +34,15 @@ def eval_N(
     Equivalent to Expression(SymbolN, expression).evaluate(evaluation)
     """
     evaluated_expression = expression.evaluate(evaluation)
-    result = eval_nvalues(evaluated_expression, prec, evaluation)
+    result = eval_NValues(evaluated_expression, prec, evaluation)
     if result is None:
         return expression
     if isinstance(result, Number):
         return result
     return result.evaluate(evaluation)
 
 
-def eval_nvalues(
+def eval_NValues(
```

Comment:
Maybe my only comment is that   `eval_NValues`  do not produce the same result as evaluating `NValues[x]` (which is return a list of `NValues`), but to apply the NValues rules associated with the elements in `x` over `x`. Maybe, adding a docstring would be enough to clarify this.

Reply:
Good point - thanks for the information. Will think about and rename.

Reply:
@mmatera - I just had a chance to look at.

Here are the only two  calls of `eval_NValues`  both in `mathics.builtin.numeric` that I see: 

```python
def eval_with_prec(self, expr, prec, evaluation, options=None):
    "N[expr_, prec_, OptionsPattern[%(name)s]]"
...
    return eval_NValues(expr, prec, evaluation)
```

and later in that file

```python
def eval_N(self, expr, evaluation: Evaluation):
    """N[expr_]"""
    # TODO: Specialize for atoms
    return eval_NValues(expr, SymbolMachinePrecision, evaluation)
```

Since eval routines return exactly the result that `NValues` returns, I think calling this `eval_NValues` is exactly the right thing to do.